src/Glpi/UI/IllustrationManager.php

```diff
@@ -109,7 +109,7 @@ public function searchIcons(
         $icons = array_filter(
             $this->getIconsDefinitions(),
             fn ($icon) => str_contains(
-                strtolower($icon['title']),
+                strtolower(_sx("Icon", $icon['title'])),
```

Comment:
Comparison should be done with the raw string, not with the sting that has its HTML entities encoded.

```suggestion
                strtolower(_x("Icon", $icon['title'])),
```